front/app/containers/AggDropDown/AggDropDown.tsx

```diff
@@ -438,8 +415,8 @@ class AggDropDown extends React.Component<AggDropDownProps, AggDropDownState> {
     const { site } = this.props;
     let display = this.props.display;
     const field = find(propEq('name', this.props.agg), [
-      ...site.search.aggs.fields,
-      ...site.search.crowdAggs.fields,
+      ...get(site, 'search.aggs.fields', []),
+      ...get(site, 'search.crowdAggs.fields', []),
```

Comment:
We're on a version of typescript that supports the optional chaining operator so this can be `search?.crowdAggs?.fields || []` without needing the lodash dependency.